packages/shared/src/components/modals/LoginModal.tsx

```diff
@@ -32,9 +37,7 @@ export default function LoginModal({
       <ModalCloseButton onClick={onRequestClose} />
       <DailyDevLogo />
       <div className="mt-6 mb-8 text-center text-theme-label-secondary typo-callout">
-        {mode === LoginModalMode.ContentQuality
-          ? `Our community cares about content quality. We require social authentication to prevent abuse.`
-          : `Unlock useful features by signing in. A bunch of cool stuff like content filters and bookmarks are waiting just for you.`}
+        {loginModalDescriptionCopy}
       </div>
       <LoginButtons />
```

Comment:
Seeing as we are already loading the flags and context here I think it would be nicer to determine the LoginButtons copy here and pass it as a child.

That way we don't have to load the context again in the child component.
What do you think?

Reply:
@rebelchris That's a good point, I'll try it.